### PR TITLE
add "Getting Started" documentation for Slackware

### PR DESCRIPTION
**Pull request by a-biardi**
Contributed: "Getting Started" documentation for Slackware

### CODE REVIEW

File: docs/Getting Started/Slackware/Root on ZFS.rst [hunk 1]
```diff
@@ -0,0 +1,232 @@
+Slackware Root on ZFS
+=====================
+
+This page shows some possible ways to configure Slackware to use zfs for the root filesystem.
+
+There are countless different ways to achieve such setup, particularly with the flexibility that zfs allows. I'm only showing you a simple recipe and giving pointers for further customization.
```
Comment (gmelikov) [inline docs/Getting Started/Slackware/Root on ZFS.rst:6]: We should use neutral way here

```suggestion
There are countless different ways to achieve such setup, particularly with the flexibility that zfs allows. We'll show only a simple recipe and give pointers for further customization.
```

File: docs/Getting Started/Slackware/Root on ZFS.rst [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+The second problem is relatively easy to deal with, and only requires slight modifications to the default Slackware initrd scripts.
+
+For the first problem, however, a variety of scenarios are possible; on a PC,
+for example, you might be booting:
+
+#. In UEFI mode, via an additional bootloader like elilo: here, the kernel and its initrd are on (read: have been copied to) the ESP, and the additional bootloader doesn't need to understand zfs.
+
+#. In UEFI mode, by booting the kernel straight from the firmware. All Slackware kernels are built with EFI_STUB=Y, so if you copy your kernel and initrd to the ESP and configure a boot entry with efibootmgr, you are all set (note that the kernel image must have a .efi extension).
+
+#. In legacy BIOS mode, using lilo or grub or similar: lilo doesn't understand zfs and even the latest grub understands it with some limitations (for example, no zstd compression). I wasn't able to get grub to work reliably. If you're stuck with legacy BIOS mode, the best option is to put /boot on a separate partition that your loader understands (for example, ext4).
```
Comment (gmelikov) [inline docs/Getting Started/Slackware/Root on ZFS.rst:36]: Let's not use `I` in official docs

File: docs/Getting Started/Slackware/Root on ZFS.rst [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+
+Outside the PC world, the only other platform supported by Slackware that I run is a Raspberry PI: in this case you have a situation similar to UEFI on the PC, where the firmware loads kernel and initrd from a filesystem that it knows about (FAT32 in this case), so again nothing to worry about.
```
Comment (gmelikov) [inline docs/Getting Started/Slackware/Root on ZFS.rst:38]: Let's not use `I` in official docs

File: docs/Getting Started/Slackware/Root on ZFS.rst [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+
+The simplest setup, discussed in this recipe, is the one using UEFI. As said above, if you boot in legacy BIOS mode, you will have to ensure that the boot loader of your choice can load the kernel image.
+
+
+Partition layout
+----------------
+
+Repartitioning an existing system disk in order to make room for a zfs root partition is left as an exercise to the reader (there's nothing specific to zfs).
+
+On my laptop, I resized the ext4 filesystem to half of available space, used sfdisk to relocate into the second half of the disk, inserted a ZFS partition before it, and copied stuff across. This approach has the benefit of providing some kind of recovery mechanism in case stuff goes wrong. When everything is okay, I then deleted the ext4 partition and enlarged the ZFS one.
```
Comment (gmelikov) [inline docs/Getting Started/Slackware/Root on ZFS.rst:48]: Let's not use `I` in official docs

File: docs/Getting Started/Slackware/Root on ZFS.rst [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+  # add more as needed:
+  # zfs create -o mountpoint=legacy [..] tank/home
+  # zfs create -o mountpoint=legacy [..] tank/usr
+  # zfs create -o mountpoint=legacy [..] tank/opt
+
+..
+
+Tweak options to taste; "mountpoint=legacy" is required for the root filesystem, and (at least on Slackware) I like to apply it to all my zfs filesystems.
+
+Also, as a personal preference, I like to set "mountpoint=none" on the pool itself so it's not mounted anywhere by default (do note that zpool's "mountpoint=none" wants an uppercase "-O").
```
Comment (gmelikov) [inline docs/Getting Started/Slackware/Root on ZFS.rst:80]: Let's not use `I` in official docs

**Review by gmelikov — COMMENTED** [review]
Thank you!

Unfortunately, I don't have comments about installation itself (I don't use slackware), so I'll point at wording mostly.

Do you ready to be mentioned on bug reports about this instruction? I propose to add info about mentioning you in issues, like https://openzfs.github.io/openzfs-docs/Getting%20Started/Debian/Debian%20Bookworm%20Root%20on%20ZFS.html#support

File: docs/Getting Started/Slackware/Root on ZFS.rst (hunk 5 above)
Comment (a-biardi) [reply docs/Getting Started/Slackware/Root on ZFS.rst]: Hi,
Sorry for the delay due to the holiday season. I have reviewed the documentation per your suggestions.
Feel free to mention my name and/or this document as you see fit.

File: docs/Getting Started/Slackware/Root on ZFS.rst [hunk 6]
```diff
@@ -0,0 +1,229 @@
+Slackware Root on ZFS
+=====================
+
+This page shows some possible ways to configure Slackware to use zfs for the root filesystem.
+
+There are countless different ways to achieve such setup, particularly with the flexibility that zfs allows. We'll show only a simple recipe and give pointers for further customization.
+
```
Comment (gmelikov) [inline docs/Getting Started/Slackware/Root on ZFS.rst:7]: Last thing then

```suggestion

Support
~~~~~~~

If you need help, reach out to the community using the :ref:`mailing_lists` or IRC at
`#zfsonlinux <ircs://irc.libera.chat/#zfsonlinux>`__ on `Libera Chat
<https://libera.chat/>`__. If you have a bug report or feature request
related to this HOWTO, please `file a new issue and mention @
a-biardi
<https://github.com/openzfs/openzfs-docs/issues/new?body=@
a-biardi,%20I%20have%20the%20following%20issue%20with%20the%Slackware%20Root%20on%20ZFS%20HOWTO:>`__.

```

**Review by gmelikov — APPROVED** [review]
Thank you!

**Pull request merged**